encrypted-media/util/utils.js

```diff
@@ -178,26 +178,34 @@ function arrayBufferAsString(buffer)
 
 function dumpKeyStatuses(keyStatuses)
 {
-    consoleWrite("for (var entry of keyStatuses)");
-    for (var entry of keyStatuses) {
-        consoleWrite(arrayBufferAsString(entry[0]) + ": " + entry[1]);
-    }
-    consoleWrite("for (var keyId of keyStatuses.keys())");
-    for (var keyId of keyStatuses.keys()) {
-        consoleWrite(arrayBufferAsString(keyId));
-    }
-    consoleWrite("for (var status of keyStatuses.values())");
-    for (var status of keyStatuses.values()) {
-        consoleWrite(status);
-    }
-    consoleWrite("for (var entry of keyStatuses.entries())");
-    for (var entry of keyStatuses.entries()) {
-        consoleWrite(arrayBufferAsString(entry[0]) + ": " + entry[1]);
+    var userAgent = navigator.userAgent.toLowerCase();
+    if ( userAgent.indexOf('edge') === -1 ) {
```

Comment:
On the first point, this would be possible, but then this debug would work only when running under the "polyfill" directory. I'd like to leave this here whilst this test is still being debugged.